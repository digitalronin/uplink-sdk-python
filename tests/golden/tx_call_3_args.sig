r = 115136800820456833737994126771386015026287095034625623644186278108926690779567
s = 85161954221260666837064522806727632936814224940955564350231565231329951451674
