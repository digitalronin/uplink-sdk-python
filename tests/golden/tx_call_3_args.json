{
    "header": {
        "tag": "TxContract",
        "contents": {
            "tag": "Call",
            "contents": {
                "address": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
                "method": "get",
                "args": [
                    {
                        "contents": 1,
                        "tag": "VInt"
                    },
                    {
                        "contents": 1,
                        "tag": "VInt"
                    },
                    {
                        "contents": 1,
                        "tag": "VInt"
                    }
                ]
            }
        }
    },
    "signature": "AE4xMTUxMzY4MDA4MjA0NTY4MzM3Mzc5OTQxMjY3NzEzODYwMTUwMjYyODcwOTUwMzQ2MjU2MjM2NDQxODYyNzgxMDg5MjY2OTA3Nzk1Njc6AE04NTE2MTk1NDIyMTI2MDY2NjgzNzA2NDUyMjgwNjcyNzYzMjkzNjgxNDIyNDk0MDk1NTU2NDM1MDIzMTU2NTIzMTMyOTk1MTQ1MTY3NA==",
    "timestamp": 1231006505,
    "origin": "fwBVDsVh8SYQy98CzYpNPcbyTRczVUZ96HszhNRB8Ve",
    "to": null
}